src/css/index.css

```diff
@@ -35,7 +35,7 @@ ol {
 }
 
 ul {
-  @apply list-disc pl-gutter space-y-2;
+  @apply pl-gutter;
 }
```

Comment:
Only issue with this is that it removes the bullets from the rest of the page content. These bullets:

<img width="1131" height="287" alt="Image" src="https://github.com/user-attachments/assets/b4027bcf-dd37-4d13-b192-8058a6903317" />

I think I found a fix:

```suggestion
main ul {
  @apply list-disc pl-gutter;
}
```